client/src/pages/components/auth/LoginScreen.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from "react";
+import Avatar from "@mui/material/Avatar";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import AuthenticationLayout from "../../../Layouts/AuthenticationLayout";
+import apiClient from "../../../utils/axios";
+import { useDispatch } from "react-redux";
+import { login } from "../../../redux/features/user/userSlice";
+import { useSelector } from "react-redux";
+import axios from "axios";
+
+import { useSnackbar } from "notistack";
+
+import { Link, useNavigate, useLocation } from "react-router-dom";
+
+const LoginScreen = () => {
+
+  const location = useLocation();
+  
+  const { enqueueSnackbar } = useSnackbar();
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const redirect = location.search ? location.search.split("=")[1] : "/";
+
+  const Login = async (e) => {
+    e.preventDefault();
+
+
+    await axios.get('http://localhost:8000/sanctum/csrf-cookie').then(async () => {
```

Comment:
please separate http requests in a separate file. you can make `src/apiRequest.js` and export it then import in this component